notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb

```diff
@@ -0,0 +1,888 @@
+    "- [Kubeflow SDK Overview](https://www.kubeflow.org/docs/components/pipelines/sdk/sdk-overview/)\n",
+    "- [Dataproc Serverless in Vertex AI Pipelines tutorial](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage3/get_started_with_dataproc_serverless_pipeline_components.ipynb)\n",
+    "- [Build a Vertex AI Pipeline](https://cloud.google.com/vertex-ai/docs/pipelines/build-pipeline)\n",
+    "\n",
+    "This notebook is built to run a Vertex AI User-Managed Notebook using the default Compute Engine Service Account.  \n",
+    "Check the Dataproc Serverless in Vertex AI Pipelines tutorial linked above to learn how to setup a different Service Account.  \n",
+    "\n",
+    "#### Permissions\n",
+    "\n",
+    "Make sure that the service account used to run the notebook has the following roles:\n",
+    "\n",
+    "- roles/aiplatform.serviceAgent\n",
+    "- roles/aiplatform.customCodeServiceAgent\n",
+    "- roles/storage.objectCreator\n",
+    "- roles/storage.objectViewer\n",
+    "- roles/dataproc.editor\n",
+    "- roles/dataproc.worker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d89b301-5249-462a-97d8-986488b303fd",
+   "metadata": {},
+   "source": [
+    "## Step 1: Install Libraries\n",
+    "#### Run Step 1 one time for each new notebook instance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "710a7dee-2c58-42a3-ae16-cb0e1d3994cd",
+   "metadata": {},
+   "source": [
+    "#### References\n",
+    "\n",
+    "- [DataprocPySparkBatchOp reference](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.0/google_cloud_pipeline_components.experimental.dataproc.html)\n",
+    "- [Kubeflow SDK Overview](https://www.kubeflow.org/docs/components/pipelines/sdk/sdk-overview/)\n",
+    "- [Dataproc Serverless in Vertex AI Pipelines tutorial](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage3/get_started_with_dataproc_serverless_pipeline_components.ipynb)\n",
+    "- [Build a Vertex AI Pipeline](https://cloud.google.com/vertex-ai/docs/pipelines/build-pipeline)\n",
+    "\n",
+    "This notebook is built to run a Vertex AI User-Managed Notebook using the default Compute Engine Service Account.  \n",
+    "Check the Dataproc Serverless in Vertex AI Pipelines tutorial linked above to learn how to setup a different Service Account.  \n",
+    "\n",
+    "#### Permissions\n",
+    "\n",
+    "Make sure that the service account used to run the notebook has the following roles:\n",
+    "\n",
+    "- roles/aiplatform.serviceAgent\n",
+    "- roles/aiplatform.customCodeServiceAgent\n",
+    "- roles/storage.objectCreator\n",
+    "- roles/storage.objectViewer\n",
+    "- roles/dataproc.editor\n",
+    "- roles/dataproc.worker"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Add a section about feedback form and feedback email id before Step 1

Reply:
added